hpicker.js

```diff
@@ -96,7 +99,16 @@ class HorizontalPicker extends Component {
     if (rangeChanged) {
       // The given children have changed
       log('rangeChanged -> resposition');
-      this.scrollToIndex(index, false);
+      if (Platform.OS === 'android') {
+        // Android fix; For some reason it only scroll a small distance. Looks
+        // pretty awful and needs a better solution, this is just to fix the
+        // out-of-sync issue.
+        setTimeout(() => {
```

Comment:
I'm curious about this. I've ignored Android a little too much, but would really like to fix this. What do you mean by it scrolling only a small distance? Does it look like the scrolling is cancelled for some reason? Adding the delay I fear will just postpone the issue to a later point.

Reply:
This was your commit that was not merged into master (https://github.com/mkko/react-native-hpicker/pull/3/commits/7dbe651bd6331e5895469de6548f7d2a00963052). I've seen some issues with rendering even with this on slow android devices. I may try to fix it later, but it is somehow acceptable for me with this delay :)

Reply:
Oh, thanks for pointing it out! I have a faint recollection of the fix but don't remember the issue with it. I don't have a physical Android device right now to test this out, but I guess it would be better to include as it is.